webapp/channels/src/components/admin_console/team_channel_settings/team/details/index.ts

```diff
@@ -17,16 +17,19 @@ import {
 } from 'mattermost-redux/actions/groups';
 
 import {setNavigationBlocked} from 'actions/admin_actions';
-
-import TeamDetails from './team_details';
-
-function mapStateToProps(state, props) {
-    const teamID = props.match.params.team_id;
-    const team = getTeam(state, teamID);
-    const groups = getGroupsAssociatedToTeam(state, teamID);
-    const allGroups = getAllGroups(state, teamID);
-    const totalGroups = groups.length;
-    const isLicensedForLDAPGroups = state.entities.general.license.LDAPGroups === 'true';
+import TeamDetails, {Props as TeamDetailsProps} from './team_details';
+import {GlobalState} from 'types/store';
+import {Team} from '@mattermost/types/teams';
+import {Group} from '@mattermost/types/groups';
+import {ActionFunc, GenericAction} from 'mattermost-redux/types/actions';
+
+function mapStateToProps(state: GlobalState, props: TeamDetailsProps) {
+    const teamID: string = props.teamID;
```

Comment:
why was this changed from `const teamID = props.match.params.team_id;` ?

Reply:
`props.match.params` is not recognized by TypeScript's transpiler after importing the `props` as `TeamDetailsProps` from the `TeamDetails` component. TypeScript's transpiler also does not recognize `props` without explicitly importing them from `TeamDetails`. Using `props.teamID` resolves the above issue.

Reply:
In the original code, `props.match.params.team_id` was used to extract the `team_id` parameter from the match object within the props. This assumes that the component is being rendered within a routing context where the `team_id` parameter is available as part of the URL.

Reply:
Reverted the code to use `props.match.params.team_id` to obtain team_jd and added an `OwnProps `object to replace `TeamDetailsProps` to pass as parameter